week2-6.2-the_2000_run.py

```diff
@@ -0,0 +1,20 @@
+import time
+
+
+def timer(f, *args, **kwargs):
+    """
+    this function receives a pointer to another function and its params,
+    the program imports time module so the function can compute the run
+    time of that function, and returns it.
+    :param f: this is a pointer to function
+    :param p: multiple parameters
+    :return: the time taken to undergo the function
+    """
+
+    start = time.time()
```

Comment:
time is not accurate enough in order to measure simple functions speed.
Read in the time module documentation whether there's a more accurate function that can be used.